code/game/objects/effects/spawners/f13lootdrop.dm

```diff
@@ -383,15 +383,47 @@
 				/obj/effect/spawner/bundle/f13/armor/combat/mk2/dark
 				)
 
-/obj/effect/spawner/lootdrop/f13/armor/tier4PA // TIER 4 ARMOR WITH PA
-	name = "tier 4 armor with possible PA"
+/obj/effect/spawner/lootdrop/f13/armor/tier4PA // Not actually tier4 but too lazy to rename/repath and remap. Use this very sparingly as it has PA.
+	name = "tier between 4 and 5 armor with possible PA"
 	loot = list(
-				/obj/effect/spawner/bundle/f13/armor/t45d,
-				/obj/effect/spawner/bundle/f13/armor/riot,
-				/obj/effect/spawner/bundle/f13/armor/combat/mk2,
-				/obj/effect/spawner/bundle/f13/armor/combat/mk2/dark
+				/obj/effect/spawner/bundle/f13/armor/advanced = 4,
```

Comment:
This armor is worse than T51b.

Reply:
If it is, I don't see the issue. It's "drip."